components/types/types.ts

```diff
@@ -340,6 +366,13 @@ export interface UserProfile {
         }
         PeacockFavoriteContracts: string[]
         PeacockCompletedEscalations: string[]
+        Saves: {
```

Comment:
This can be null for existing profiles, null checks are needed wherever this is used.